source/server-discovery-and-monitoring/tests/integration/minPoolSize-error.yml

```diff
@@ -0,0 +1,79 @@
+# Test SDAM error handling.
+runOn:
+    # failCommand appName requirements
+    - minServerVersion: "4.4"
+
+database_name: &database_name "sdam-tests"
+collection_name: &collection_name "sdam-minPoolSize-error"
+
+data: []
+
+tests:
+  - description: Command error on minPoolSize background creation
+    # Configure the initial monitor handshake to succeed but the
+    # first or second background minPoolSize establishments to fail.
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { skip: 2 }
+      data:
+        failCommands: ["isMaster"]
+        appName: SDAMminPoolSizeError
+        closeConnection: false
+        errorCode: 91 # ShutdownInProgress
```

Comment:
When this test uses a command error, the driver does not cancel the inprogress streaming isMaster which messes up the assertions in this test. I suggest we change this to be `closeConnection: true`.

Reply:
Fixed